var/spack/repos/builtin/packages/py-pyprecice/package.py

```diff
@@ -15,22 +15,29 @@ class PyPyprecice(PythonPackage):
     homepage = "https://www.precice.org"
     git = "https://github.com/precice/python-bindings.git"
     url = "https://github.com/precice/python-bindings/archive/v2.0.0.1.tar.gz"
-    maintainers = ["ajaust", "BenjaminRueth"]
+    maintainers = ["ajaust", "BenjaminRodenberg"]
 
     # Always prefer final version of release candidate
     version("develop", branch="develop")
+    version('2.2.0.1', sha256='032fa58193cfa69e3be37557977056e8f507d89b40c490a351d17271269b25ad')
+    version('2.1.1.2', sha256='363eb3eeccf964fd5ee87012c1032353dd1518662868f2b51f04a6d8a7154045')
     version("2.1.1.1", sha256="972f574549344b6155a8dd415b6d82512e00fa154ca25ae7e36b68d4d2ed2cf4")
     version("2.1.0.1", sha256="ac5cb7412c6b96b08a04fa86ea38e52d91ea739a3bd1c209baa93a8275e4e01a")
     version("2.0.2.1", sha256="c6fca26332316de041f559aecbf23122a85d6348baa5d3252be4ddcd5e94c09a")
     version("2.0.1.1", sha256="2791e7c7e2b04bc918f09f3dfca2d3371e6f8cbb7e57c82bd674703f4fa00be7")
     version("2.0.0.2", sha256="5f055d809d65ec2e81f4d001812a250f50418de59990b47d6bcb12b88da5f5d7")
     version("2.0.0.1", sha256="96eafdf421ec61ad6fcf0ab1d3cf210831a815272984c470b2aea57d4d0c9e0e")
 
-    patch("deactivate-version-check-via-pip.patch")
+    # Import module as a test
+    import_modules = ["precice"]
 
-    variant("mpi", default=True, description="Enables MPI support")
+    # Older versions of the bindings checked versions via pip. This patch
+    # removes the pip dependency.
+    # See also https://github.com/spack/spack/pull/19558
+    patch("deactivate-version-check-via-pip.patch", when="@:2.1.1.1")
 
-    depends_on("mpi", when="+mpi")
+    depends_on("precice", when="@develop")
```

Comment:
Should this require `precice@develop`?

Reply:
I think it hat is for @BenjaminRodenberg. Does the `develop` branch depend on the latest preCICE release or preCICE's `develop` branch?

Reply:
On `develop`. So it should indeed be `depends_on("precice@develop", when="@develop")`.

Reply:
I have updated the Spack recipe accordingly.